main.tf

```diff
@@ -119,24 +111,20 @@ resource "google_cloud_run_service" "service" {
 resource "google_cloud_run_domain_mapping" "domain_mapping" {
   count = var.module_enabled && var.domain_mapping != null ? 1 : 0
 
-  name     = var.domain_mapping.name
-  location = var.domain_mapping.location
-  project  = var.project
+  name     = google_cloud_run_service.service[0].name
+  location = google_cloud_run_service.service[0].location
+  project  = google_cloud_run_service.service[0].project
```

Comment:
is this always will be `google_cloud_run_service`'s attributes? can't i map something different (as before from the variable)?